jbang-catalog.json

```diff
@@ -0,0 +1,8 @@
+{
+  "aliases": {
+    "jython-cli": {
+      "script-ref": "jython_cli.java",
+      "description": "Run Jython 2.7.4"
```

Comment:
Just a thought: Maybe we can just write 2.7 so this description would not require frequent updating.

Reply:
Fixed.